YearInReview/Model/Reports/ReportGenerator.cs

```diff
@@ -55,8 +58,26 @@ public async Task<Report1970> Generate(int year)
 			var games = _playniteApi.Database.Games;
 			var activities = await _gameActivityExtension.GetActivityForGames(games);
 			var filteredActivities = _specificYearActivityFilter.GetActivityForYear(year, activities);
+			
+			ShowReportGeneratedNotification(year);
 
 			return _composer1970.Compose(year, filteredActivities);
 		}
+
+		private void ShowReportGeneratedNotification(int year)
+		{
+			var description = string.Format(
+				ResourceProvider.GetString("LOC_YearInReview_ReportGeneratedNotification"), 
+				year
+			);
+			
+			_playniteApi.Notifications.Add(
+				new NotificationMessage(
+					NotificationId + year,
```

Comment:
I usually keep same notification id for all the notifications of same type. Then if we get multiple notifications only the last one would be shown.